traffic_portal/app/src/modules/private/deliveryServices/staticDnsEntries/list/index.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+module.exports = angular.module('trafficPortal.private.deliveryServices.staticDnsEntries.list', [])
+	.config(function($stateProvider, $urlRouterProvider) {
+		$stateProvider
+			.state('trafficPortal.private.deliveryServices.staticDnsEntries.list', {
+				url: '',
+				views: {
+					deliveryServicesStaticDnsEntriesContent: {
+						templateUrl: 'common/modules/table/deliveryServiceStaticDnsEntries/table.deliveryServiceStaticDnsEntries.tpl.html',
+						controller: 'TableDeliveryServiceStaticDnsEntriesController',
+						resolve: {
+							deliveryService: function($stateParams, deliveryServiceService) {
+								return deliveryServiceService.getDeliveryService($stateParams.deliveryServiceId);
+							},
+							staticDnsEntries: function($stateParams, deliveryServiceStaticDnsEntryService) {
+								return deliveryServiceStaticDnsEntryService.getStaticDnsEntries({ deliveryserviceId: $stateParams.deliveryServiceId });
```

Comment:
so here you refer to `deliveryServiceStaticDnsEntryService` but in traffic_portal/app/src/common/api/index.js you have named it `staticDnsEntryService`